auth/src/desktop/auth_providers/facebook_auth_credential.h

```diff
@@ -31,9 +32,9 @@ class FacebookAuthCredential : public IdentityProviderCredential {
   std::string GetProvider() const override { return kFacebookAuthProviderId; }
 
   std::unique_ptr<VerifyAssertionRequest> CreateVerifyAssertionRequest(
-      const char* const api_key) const override {
+      ::firebase::App& app, const char* const api_key) const override {
```

Comment:
I tried that, but kept App as non-const since these subclasses of AuthRequest can call GetAndClearHeartbeats which is a non const method since it can mutate state in the internal HeartbeatController.